iroha/src/smartcontracts/isi/expression.rs

```diff
@@ -10,22 +10,23 @@ use iroha_error::{error, Error, Result};
 
 use super::Evaluate;
 use crate::prelude::*;
+use crate::wsv::WorldTrait;
```

Comment:
Please combine imports into one.

Reply:
You can pick an option, which should be fine. You can make a separate pr with format fix.
https://rust-lang.github.io/rustfmt/?version=v1.4.36&search=use#imports_granularity

Reply:
Let's select `Crate` fmt setting then https://rust-lang.github.io/rustfmt/?version=v1.4.36&search=use#Crate%5C%3A

Reply:
https://jira.hyperledger.org/browse/IR-1148